src/DynamoPackagesWPF/PackageManagerViewExtension.cs

```diff
@@ -138,6 +138,9 @@ private void RequestLoadViewExtensionsForLoadedPackages(IEnumerable<Package> pac
 
         private void RaisePackageHostNotifications(IEnumerable<Package> packages)
         {
+            // ALIAS does needs a total network traffic blackout.
+            // Since there is no way to switch this on/off, we will just skip it for now.
+            /*
             foreach (var pkg in packages)
```

Comment:
This code would not raise network traffic right? This is the code to check if the installed package is targetting other host. I think @zeusongit recently worked on it..

Reply:
I commented it out because it did try to contact pacakage manager.
I could not find a way to disable through API or configs. @zeusongit any idea if there is a way to disable this ?

Reply:
After reading the code, I can see why now. In function `CheckIfPackagesTargetOtherHosts` there is a call `var knownHosts = PackageManagerClient.GetKnownHosts();` which will contact package manager

Reply:
I think this solution is fine for now but we probably want to guard all of these code using the env variable or setting property on `master` branch.